Grid/qcd/hmc/GenericHMCrunner.h

```diff
@@ -47,6 +47,9 @@ template <class Implementation,
           class RepresentationsPolicy = NoHirep, class ReaderClass = XmlReader>
 class HMCWrapperTemplate: public HMCRunnerBase<ReaderClass> {
 public:
+  // The with LLR boolean switch
+  bool with_llr = false;
```

Comment:
I don't believe that there are are any situations where we want to take an HMC from being LLR to being non-LLR or vice-versa, so this should probably be `const`.